Makefile

```diff
@@ -72,6 +72,9 @@ up/basic: up/essential
 .PHONY: up/essential
 up/essential: get vendor/hosts
 	@$(foreach SVC, $(START_ESSENTIAL), $(shell docker-compose -f services/$(SVC)/docker-compose.yml up -d))
+	@./vendor/neofs-adm --config services/ir/neofs-adm.yml morph init --alphabet-wallets ./services/ir --contracts vendor/contracts
```

Comment:
Why we don't fail here?

Reply:
> Why we don't fail here?

Not sure what are reasons to fail here.

> Also, alphabet-wallets can be put in the config file.

I think it was intentional decision by @carpawell to avoid relative paths in config, so one can run neosf-adm from anywhere.

Reply:
If for some reasons we can't `init`, one example is MaxGASInvoke, which you have fixed, but contract/adm update can bring others, especially on nightly.

Reply:
Now we fail with `|| exit 1` on `init` and `transfer` commands.